tools/project_manager/show_project_possible_bad_words.php

```diff
@@ -107,11 +107,11 @@
 $checkbox_form["freqCutoff"]=$freqCutoff;
 echo_checkbox_selects(count($bad_words_w_freq));
 echo_checkbox_form_start($checkbox_form);
-echo_checkbox_form_submit(_("Add selected words to Bad Words List"));
+echo_checkbox_form_submit(_("Add selected words to Bad Words List"), BAD_WORD_LIST);
```

Comment:
The possibility exists that a PM might want to add a word to the GWL that is identified as a possible bad word. If I looked at the following, I'd just as soon add "tip" to the GWL and not bother the proofers with worrying about it.

<img width="275" alt="image" src="https://user-images.githubusercontent.com/36800239/80329957-79eed580-8809-11ea-984d-c9698e811342.png">

and 

<img width="304" alt="image" src="https://user-images.githubusercontent.com/36800239/80329990-9428b380-8809-11ea-858b-c7d282bba7f7.png">

I realize that the task really deals only with the GWL, and this could be considered scope creep -- should it be a different task?

Reply:
Yes, that's scope creep and we should punt that. I've already scope-creeped this a bit much already for poor Chris.

Reply:
<grin> Understood.

It should be documented, then, and perhaps a new task created after it's rolled out. I could always be wrong, but this strikes me as the sort of inconsistency that will be noticed and questioned.

Reply:
Create the ticket. This shouldn't be hard to add. Just trying to not go past the upcoming code freeze.